tools/msgfplus/msgfplus.xml

```diff
@@ -82,7 +85,13 @@
                       label="Collection: Pairs of spectra (forward) and FASTA database (reverse)"/>
               </when>
         </conditional>
-        <param name="tsvcheck" type="boolean" label="Output TSV as well?" />
+        <conditional name="tsv">
```

Comment:
Why do you use a conditional here, it could be a single boolean isn't it?

Reply:
It's to make the showDecoy boolean appear only if produce TSV is checked. Wrong UI?

Reply:
Ok got it, I think this is a little bit confusing. Why not a simple select, "TSV outpout", "TSV ooutput with decoys" ...